packages/MarkdownEditor-Tests.package/MarkdownInlineTextStylerTest.class/instance/testDelimiterCantCloseItself.st

```diff
@@ -20,6 +20,7 @@ testDelimiterCantCloseItself
 				(MarkdownString new content: 'b'; yourself)}.
 	
 	inlineTextStyler findOpenerFor: openCloser 
-		ifFound: [:o | foundOpener := o].
-	self deny: (openCloser = foundOpener).
-	self assert: opener equals: foundOpener
\ No newline at end of file
+		ifFound: [:foundOpener | 
+			self deny: (openCloser = foundOpener).
```

Comment:
Isn't this `deny` redundant? We know that `opener` and `openCloser` are two variables, and when we assert that the `foundOpener` is `opener`, it can't be `openCloser` by transitivity, right?

Reply:
We just left it in. In my point of view, this "redundancy" makes the test easier to understand. From a logical standpoint, you're absolutely right. But pointing out, that it is not the `openCloser` but the `opener` adds some clarity.

Reply:
Okay, in my case it was the opposite, because I have both of a positive and a negative assert and we actually want to test equality. As a reader, I need to decide whether the deny or the assert is the important part. Thus, it doesn't make it clearer for me, but confuses me...